pydantic/types.py

```diff
@@ -713,7 +713,7 @@ class Model(BaseModel):
 path.unlink()
 
 path = Path('directory')
-path.mkdir()
+path.mkdir(exist_ok=True)
```

Comment:
Unrelated to this PR, but this resolved some local test failures for me, and I don't see any reason not to keep it there.